tests/jobs/test_backfill_job.py

```diff
@@ -630,11 +630,11 @@ def test_backfill_rerun_upstream_failed_tasks(self):
             schedule_interval='@daily')
 
         with dag:
-            t1 = DummyOperator(task_id='test_backfill_rerun_upstream_failed_task-1',
-                               dag=dag)
-            t2 = DummyOperator(task_id='test_backfill_rerun_upstream_failed_task-2',
-                               dag=dag)
-            t1.set_upstream(t2)
+            task_1 = DummyOperator(task_id='test_backfill_rerun_upstream_failed_task-1',
```

Comment:
Well, this makes me think that you have to rebase on new master because I have already fixed all pylints in tests.

Reply:
Yeah!